packages/web-components/src/tree-item/tree-item.style.ts

```diff
@@ -0,0 +1,220 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusMedium,
+  colorNeutralForeground2,
+  colorNeutralForeground2Hover,
+  colorNeutralForeground2Pressed,
+  colorNeutralForeground2Selected,
+  colorNeutralForeground3,
+  colorNeutralForeground3Hover,
+  colorNeutralForeground3Pressed,
+  colorNeutralForeground3Selected,
+  colorStrokeFocus2,
+  colorSubtleBackground,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundLightAlphaHover,
+  colorSubtleBackgroundLightAlphaPressed,
+  colorSubtleBackgroundLightAlphaSelected,
+  colorSubtleBackgroundPressed,
+  colorSubtleBackgroundSelected,
+  colorTransparentBackground,
+  colorTransparentBackgroundHover,
+  colorTransparentBackgroundPressed,
+  colorTransparentBackgroundSelected,
+  curveEasyEaseMax,
+  durationFaster,
+  fontSizeBase300,
+  spacingHorizontalM,
+  spacingHorizontalXS,
+  spacingHorizontalXXL,
+  spacingHorizontalXXS,
+  spacingVerticalNone,
+  spacingVerticalS,
+  spacingVerticalXXL,
+  spacingVerticalXXS,
+  spacingVerticalXXXL,
+} from '../theme/design-tokens.js';
+
+export const treeItemLevelToken = '--fluentTreeItemLevel' as const;
+
+export const styles = css`
+  :host {
+    display: block;
+    outline: none;
+  }
+
+  :host(:focus-visible) .positioning-region {
+    box-shadow: ${spacingVerticalNone} ${spacingVerticalNone} ${spacingVerticalNone} ${spacingVerticalXXS}
+      ${colorStrokeFocus2} inset;
+  }
+  @media (prefers-contrast: more) {
+    :host(:focus-visible) .positioning-region {
+      outline: 1px solid ${colorStrokeFocus2};
+    }
+  }
+
+  /**
+   * Default variants:
+   * Size - medium
+   * Appearance - subtle
+   */
+  .positioning-region {
+    display: flex;
+    align-items: stretch;
+    justify-content: space-between;
+    cursor: pointer;
+
+    height: ${spacingVerticalXXXL};
+    padding-inline-end: ${spacingVerticalS};
+    border-radius: ${borderRadiusMedium};
+    background-color: ${colorSubtleBackground};
+    color: ${colorNeutralForeground2};
+
+    /**
+     * The positioning-region's font size is inherited from host element,
+     * therefore:
+     *
+     * indentation: 0
+     *     indentation: 24
+     *         indentation: 48
+     *             ...
+     */
+    padding-inline-start: calc(var(${treeItemLevelToken}) * ${spacingHorizontalXXL});
+
+    & .content-region {
```

Comment:
I'm not sure if CSS Nesting's browser support is where we want it to be, e.g. we generally support Chromium down to 100 and Safari down to 16.4, but CSS Nesting is supported in neither.